src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -91,6 +91,18 @@ case class MethodConfigurationNamePair(
                    destination: MethodConfigurationName
                    )
 
+@ApiModel(value = "Entity copy definition")
+case class EntityCopyDefinition(
+                   @(ApiModelProperty@field)(required = true, value = "The source workspace of the entities")
+                   sourceWorkspace: WorkspaceName,
+                   @(ApiModelProperty@field)(required = true, value = "The destination workspace for the entities")
+                   destinationWorkspace: WorkspaceName,
```

Comment:
the main use now is copying from a "vault"y workspace into your workspace, but can be used to copy within the same workspace